values.yaml

```diff
@@ -38,6 +38,15 @@ server:
   replicas: 3
   bootstrapExpect: 3 # Should <= replicas count
 
+  # extraEnvVars is a list of extra enviroment variables to set with the stateful set. These could be
+  # used to include proxy settings required for cloud auto-join feature, 
+  # in case kubernetes cluster is behind egress http proxies. Additionally, it could be used to configure
+  # custom consul parameters.
+  extraEnvironmentVars: {}
```

Comment:
To maintain consistency in variable ordering with the `client` section, this would fit better coming after the `extraVolumes` option under `server`.